cassandra-commons/src/test/java/com/mesosphere/dcos/cassandra/common/tasks/cleanup/CleanupContextTest.java

```diff
@@ -27,7 +30,7 @@ public void testJSONSerializationWithSnakeCaseMembers() throws Exception {
 
         Assert.assertEquals(Arrays.asList("column_families", "key_spaces", "nodes"), keys);
 
-        context = JsonUtils.MAPPER.readValue(jsonContext, CleanupContext.class);
+        context = serializer.deserialize(jsonContext.getBytes(StandardCharsets.ISO_8859_1), CleanupContext.class);
```

Comment:
Just because it was being used in the prior version of these files. I'll switch it to UTF8 in a separate PR since there's no other reason for having it like that.